### ISSUE TITLE
Character Interactions Broken On Android, Auto Selecting Cards Soft Locks Your Game, Song Pause not working, Last Song Played Not Saving (Android and PC) Micro Stuttering on Higher End Android Pones?

### ISSUE THREAD

**Issue by SirTankBall** [Xele02/UMO#157]
srry about the 20 some min video but this is the best i can do and show all the bugs ive seen 

https://youtu.be/8WH21aG2uus

time stamps for all the bugs seen 

[0:00] [1:12] - Character Interactions Broken On Android.
[1:12] - [4:50] - Auto Selecting Cards Soft Locks Your Game.
[4:50] - [7:30] - Screen Rotate 180 When Starting Song/Pause Song Not Working.
[7:30] - [14:06] - Micro Stuttering on Higher End Android Phones?
[14:07] - [20:13] - Last Song Played Not Saving (Android and PC)

**Comment by Tiger1970prod** [comment]
Menu last played issue already reported:
https://github.com/Xele02/UMO/issues/144

**Comment by SirTankBall** [comment]
> Menu last played issue already reported: #144

yes i know. i added it for the YT video so ppl that r just seeing this for the first time know what to expect.

**Comment by Xele02** [comment]
> > Menu last played issue already reported: #144
> 
> yes i know, but its a good thing to have more then one person having a bug show up. i added it for the YT video so ppl that r just seeing this for the first time know what to expect.

Yeah, but it's easier for me to have one issue by bug (eaiser to track which one is fixed), and if the bug already exist, add more info in comment. But thanks anyway for the report.
